### PR TITLE
Add contact form validation

### PR DESCRIPTION
**Pull request by FilbertKimasa**
In this pull request, the following was done;

- [ ] Add contact form validation

### CODE REVIEW

**Review by bobb-Rob — COMMENTED** [review]
Hi @FilbertKimasa , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that should show commit messages from your coding partner since this is a pair programming activity. Could you please request your coding partner to make substantial contributions to the project by providing commit messages for the pair programming activity? The project aims to showcase the collaborative efforts and achievements of both partners.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by ademibowale — COMMENTED** [review]
Hi @majdalkilany ,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request




Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

You have submitted a project, that should show commit messages from your coding partner since this is a pair programming activity. Could you please request your coding partner to make substantial contributions to the project by providing commit messages for the pair programming activity? The project aims to showcase the collaborative efforts and achievements of both partners.

Your Code R

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: app.js [hunk 1]
```diff
@@ -547,3 +547,17 @@ if (desktopCards) {
...
+const errorDiv = document.getElementById('error-message');
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+  const email = document.querySelector('.input-email').value;
+  if (email === email.toLowerCase()) {
+    form.submit();
+  } else {
+    errorDiv.innerText = 'The email must be in lowcase';
+  }
+});
```
Comment (souad988) [inline app.js]: Great work by implementing this functionality it works just as expected 🥇 
[OPTIONAL] one small thing to add is removing the error message when the user changes the input content as it is confusing to keep it while a valid email is entred this is important for good UI

Capture d’écran du 2023-08-05 10-59-04

**Review by souad988 — APPROVED** [review]
Hi @FilbertKimasa  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

approved2


## Highlights
DRY js code
Show error message on non valid email

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**